plugins/gpgme-vala/vapi/gpgme.vapi

```diff
@@ -505,11 +505,13 @@ namespace GPG {
     [CCode (cname = "gpgme_get_engine_information")]
     public GPGError.Error get_engine_information(out EngineInfo engine_info);
 
-    [CCode (cname = "gpgme_strerror_r")]
-    public int strerror_r(GPGError.Error err, uint8[] buf);
+    public int strerror_r(GPGError.Error err, uint8[] buf) {
+        return err.strerror_r(buf);
+    }
 
-    [CCode (cname = "gpgme_strerror")]
-    public unowned string strerror(GPGError.Error err);
+    public unowned string strerror(GPGError.Error err) {
+        return err.to_string();
+    }
```

Comment:
tbh I forget why I did this.  I assume there was a good reason I needed to flesh out these functions instead of just leaving them as with that `CCode` decorator.